info-creator/io500_info_editor.py

```diff
@@ -167,14 +167,24 @@ def edit_infos(sitefile, tokens, schemafile = "schema-io500.json"):
       json.dump(data, f, indent=2)
       f.truncate()
 
+def execute_download(url, outfile):
+  import subprocess
+  try:
```

Comment:
This would be a good place to add the prompt, like:

   if (tty)
       OK to download <url>? [Y/n]